### PR TITLE
silhouette score

### PR DESCRIPTION
**Pull request by lewisExternal**
Hey @Steeeephen 

Following up on the recent Data Science Festival event, thanks again for the talk. 
As mentioned, adding an example of silhouette score as an intrinsic validator for clustering techniques. 
I did have to update one of the packages but I also had to change the rapids version to account for different hardware, so please validate the environment resolves before merging. 

Just a thought! 

REF 
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.silhouette_score.html
https://towardsdatascience.com/silhouette-coefficient-validating-clustering-techniques-e976bb81d10c

### CODE REVIEW

File: buzzwords/buzzwords.py [hunk 1]
```diff
@@ -594,3 +601,19 @@ def load(self, destination: str) -> None:
 
         with open(destination, 'rb') as file:
             self.__dict__ = pickle.load(file)
+
+    def get_silhouette_score(self, X: np.ndarray, labels: np.ndarray) -> float:
```
Comment (Steeeephen) [inline buzzwords/buzzwords.py:608]: says it returns `float` but it actually returns `NoneType` from the print

think it's more intuitive to keep this as `-> float` and change the `print` to a `return`, then printing after calling it

**Review by Steeeephen — CHANGES_REQUESTED** [review]
Really nice change, I like it. One small request outside of the comment - make it optional to print the output, not every end user may want to add it to their logs

**Comment by lewisExternal** [comment]
Thanks @Steeeephen, all good points - these should now be resolved. 
Anything else, please do let me know.

**Review by Steeeephen — APPROVED** [review]
This is perfect, I really like it. Thanks so much for contributing! 🥺

**Comment by lewisExternal** [comment]
> This is perfect, I really like it. Thanks so much for contributing! 🥺

Thanks for putting code into the wild and for the event, pizza for PRs seems like a fair exchange to me.

1843_20210415_HAWAIIANPIZZA_001

**Pull request merged**